web-client/src/views/Public/TrialSessions/PublicTrialSessionDetails.tsx

```diff
@@ -245,18 +245,22 @@ function MobileOpenCases({
                         publicCase.privatePractitioners.length > 0 && (
                           <div>
                             <span className="label">Petitioner Counsel</span>
-                            {publicCase.privatePractitioners?.map(counsel => (
-                              <div key={counsel.name}>{counsel.name}</div>
-                            ))}
+                            <div className="display-flex flex-column gap-1">
```

Comment:
While I'm here, adding a tiny bit of spacing between names just in case there is a really long one.